app/controllers/post-settings-menu.js

```diff
@@ -292,50 +293,57 @@ export default Controller.extend(SettingsMenuMixin, {
             if (!userInput) {
                 // Clear out the publishedAt field for a draft
                 if (this.get('model.isDraft')) {
-                    this.set('model.publishedAt', null);
+                    this.set('model.publishedAtOffset', null);
```

Comment:
`publishedAtOffset` is a computed property so you don't want to set it directly (it will overwrite the CP).

If you want `publishedAtOffset` to return `null` I would suggest ensuring it returns null and doesn't error when `publishedAt` is `null`, then reverting this change so that we continue clearing the `publishedAt` value.